Microsoft.Identity.Web/WebApiServiceCollectionExtensions.cs

```diff
@@ -99,6 +99,57 @@ public static class WebApiServiceCollectionExtensions
 
         // TODO: pass an option with a section name to bind the options ? or a delegate?
 
+        public static IServiceCollection AddProtectedWebApiB2C(
```

Comment:
i'm not sure why this needs to have B2C in it...isn't it the same as calling any protected web api. this is not b2c specific.